Week_6/sentimental-cash/cash.py

```diff
@@ -0,0 +1,28 @@
+def main():
+    while True:
+         change = input("How much change is owned: ")
+         if change < 0:
+            break
+    convertCents = round(change * 100)
+    coins = 0
+
+    while convertCents >= 25:
+        convertCents = convertCents - 25
+        coins += 1
+    
+    while convertCents >= 10:
+        convertCents = convertCents - 10
+        coins += 1
+    
+    while convertCents >= 5:
+        convertCents = convertCents - 5
+        coins += 1
+    
+    while convertCents >= 1 == convertCents > 0:
+        convertCents = convertCents - 1
+        coins += 1
+    
+    print(f"%d\n", coins)
+    
+    main()
+    
```

Comment:
Missing EOF

Reply:
Instead of calling `main()` again, you could put the content of the function in a `while True` loop.